docs/App.fsproj

```diff
@@ -28,6 +28,7 @@
         <Compile Include="Recharts/BarCharts/MixBarChart.fs" />
         <Compile Include="Recharts/BarCharts/TinyBarChart.fs" />
         <Compile Include="Recharts/BarCharts/PositiveAndNegative.fs" />
+        <Compile Include="Recharts\PieCharts\CustomizedLabelPieChart.fs" />
```

Comment:
```suggestion
        <Compile Include="Recharts/PieCharts/CustomizedLabelPieChart.fs" />
```

Reply:
Sorry I guess I merged before checking your review. I will change the slashes myself before I publish